R/type-tibble.R

```diff
@@ -1,5 +1,10 @@
 # Coercion ----------------------------------------------------------------
 
+new_tibble <- function(df) {
```

Comment:
I think calling this `new_tibble()` is potentially confusing because it's not really a constructor.

Reply:
Good point, it is now `df_as_tibble()`.